### ISSUE TITLE
node-haste status?

### ISSUE THREAD

**Issue by colinrcummings** [facebook/react#9535]
In the [codebase overview](https://facebook.github.io/react/contributing/codebase-overview.html) there's still reference to Haste; however, I see elsewhere that it's been [unsupported/archived](https://github.com/facebookarchive/node-haste/tree/archived) for several months. Have you already migrated to CommonJS / ES Modules? I'd like to pick up an open issue, but wanted to validate first. Thanks!

**Comment by gaearon** [comment]
Hi! I don’t know what `node-haste` repo is, but we still use Haste (it’s more of a convention than a specific tool really).

We do plan to migrate to ES modules, but we’re waiting on some internal fixes to get through first.
 I’ll close as there was another issue about dropping Haste before.
